shared/actions/platform-specific/index.native.js

```diff
@@ -203,20 +202,27 @@ function* persistRoute(state, action) {
   }
 
   const path = action.payload.path
-  const top = path[path.length - 1]
-  if (!top) return
+  const tab = path[path.length - 2] // real top is the root of the tab (aka chatRoot) and not the tab itself
```

Comment:
I don't understand this logic. How will it always be true that the tab will the be penultimate item?

Reply:
updated